### PR TITLE
feat(Docs): Development guide

### PR DESCRIPTION
**Pull request by stephenburgess8**
Hey - very rough first draft with some more detailed instructions to help with first time contributors. Feel free to jump in and edit yourself if you want to make some changes. Otherwise open to suggestions, corrections, or different ways of doing things. Ideally eventually this would also have Linux and Windows specific instructions. Also eventually it would be nice to break this out into a Development guide as well as a Contributing guide. Once this is in good shape, we can remove some of the duplicate information from the README.md and maybe just add a link.

Besides just having a little more detail, the main difference between this and the existing instructions are that I needed to set the `--database-url='postgres://localhost/plume'`for `diesel setup` and `diesel migration`.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Now that the Postgres database is set up and the `plume` user has the privileges it  needs, we can set up the database using the diesel CLI. If this was your time installing Rust, you will probably need to run that using `cargo`. `cargo` is installed with `rustc` so if you followed the earlier instructions it will already be available.
+```
+cargo install diesel_cli
+```
+The first time you run this, you can run setup. After that, every time you pull the repository you will want to run the migration command in case there were any migrations. Those commands are
+```
+diesel setup --database-url='postgres://localhost/plume'
+diesel migration run --database-url='postgres://localhost/plume'
+```
+#### Configuration
```
Comment (BaptisteGelez) [inline DEVELOPMENT.md]: I think this part should be moved after "Running Plume"

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+```
+git remote add upstream https://github.com/myname/Plume.git
+```
+Now, make any changes to the code you want. After committing your changes, push to the upstream fork. Once your changes are made, visit the GitHub page for your fork and select "Make Pull Request". Add descriptive text, any issue numbers using hashtags to reference the issue number, screenshots of your changes if relevant, a description of how you tested your changes, and any other information that will help the project maintainers be able to quickly accept your pull requests.
+The project maintainers may suggestion further changes to improve the pull request even more. After implementing this locally, you can push to your upstream fork again and the changes will immediately show up in the pull request after pushing. Once all the suggested changes are made, the pull request may be accepted. Thanks for contributing.
```
Comment (BaptisteGelez) [inline DEVELOPMENT.md]: I don't know if you wanted to create a new paragraph here, but if you did, you should make sure to add a blank line between the two paragraphs.

**Comment by BaptisteGelez** [comment]
> Besides just having a little more detail, the main difference between this and the existing instructions are that I needed to set the `--database-url='postgres://localhost/plume'` for `diesel setup` and `diesel migration`

Yes, when I wrote the instructions in the README, we had a `.env` file to tell diesel where to find the database, but we deleted it to use our own environment variables and make it easier to test the federation locally.

**Comment by stephenburgess8** [comment]
Cool!. Removing the content from README.md leaves it feeling a little light. I'm trying to think of more stuff to put there. Je voudrais polir le UI suivante. I would like to work on bringing some polish to the UI next if you think that's a good idea. After that we could add some screenshots.

**Comment by BaptisteGelez** [comment]
Yes, the README is a bit short now. But that's not a real problem I think :man_shrugging: 

If you want to work on the UI, keep in mind that that a redesign have been started in #31, so it may be better to wait for this branch to be merged before (the requested changes should be made soon, normally, so it will probably be merged in a few days).

**Pull request merged**